dodo.py

```diff
@@ -349,25 +349,27 @@ def task_build():
         )
         sdist = py_pkg / f"""dist/{py_name.replace("_", "-")}-{D.PY_VERSION}.tar.gz"""
 
-        actions = [U.do("python", "setup.py", "sdist", "bdist_wheel", cwd=py_pkg)]
+        pyproj_toml = py_pkg / "pyproject.toml"
 
         file_dep = [
-            *py_pkg.rglob("src/*.py"),
+            *py_pkg.rglob("*.py"),
             *py_pkg.glob("*.md"),
             setup_py,
+            pyproj_toml,
         ]
 
         if py_name == "jupyterlite-core":
             file_dep += [B.PY_APP_PACK]
 
-        pyproj_toml = py_pkg / "pyproject.toml"
-
         targets = [wheel, sdist]
+        actions = []
+        # make sure to build the lab extension before building the wheel
+        if py_name == C.JS_KERNEL_NAME:
+            actions += [U.do("yarn", cwd=py_pkg)]
```

Comment:
if keeping this as a single job, would need to add all of the `.ts`, `.json`, `.css`, etc.

ideally in CI it would use `--frozen-lockfile`.